renovate.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended"
+  ],
+  "packageRules": [
+    {
+      "enabled": false,
+      "packagePatterns": [
+        "*"
+      ]
+    },
+    {
+      "enabled": true,
+      "matchDepPatterns": [
+        "automattic|wpmreleasetoolkit"
+      ],
+      "separateMajorMinor": false
+    },
```

Comment:
Question (❓): With this configuration now, we want to only enable dependency updates for anything that is matching the `automattic` or `wpmreleasetoolkit` pattern, but also to only create one PR for a dependency update, not separating it into multiple major/minor PRs.

This is what causing Renovate to create the below 3 (potential) PRs, correct me if I am wrong:
- Update `automatticTracksVersion` to v5.1.0
- Update buildkite plugin `automattic/a8c-ci-toolkit` to v3
- Update dependency `fastlane-plugin-wpmreleasetoolkit` to v11

Also, what about:
- The `automatticAboutVersion` version, why is that listed as a potential PR?
- How about the `automatticRestVersion` version?
- What about the `wordpress` lib related versions, like:
  - `wordPressAztecVersion`
  - `wordPressFluxCVersion`
  - `wordPressLoginVersion`
  - `wordPressPersistentEditTextVersion`
  - `wordPressUtilsVersion`
- What about some other lib versions, like:
  - `gutenbergMobileVersion`
  - `gravatarVersion`

As such, what if we include as was at lest the `wordpress` keyword into the match dependency pattern there? 🤔

Reply:
To see context of the decisions of Renovate, please see the log https://developer.mend.io/github/wordpress-mobile/WordPress-Android/-/job/ce04a4bb-215a-4343-abdc-75147cf91e8f

Some answers:
> The automatticAboutVersion version, why is that listed as a potential PR?

Because the project uses the already highest available version (`1.4.0`)

> automatticRestVersion

Same as above (btw I wonder if this dependency is still used 😅 )

> What about the wordpress lib related versions, like:

These are indeed disabled, and they shouldn't be. I'll update the configuration, thanks!

Reply:
Cool, thanks for the clarification on all that, of course the other 2 `automattic` versions are already up-to-date, didn't think to check, assumed that everything is out-of-date... 😅 

Thanks @wzieba ! 🥇

Reply:
It's strange though as I see no mention of `fluxc` in the logs 🤔

Reply:
Maybe because the current `trunk-a9a471914d3ebd1093986d3d0a95c2a29b29dca0` version is ahead of the latest [2.80.0](https://github.com/wordpress-mobile/WordPress-FluxC-Android/releases/tag/2.80.0) tag? 🤔

Reply:
Btw, another one, how about `dangermattic` in addition to having `wpmreleasetoolkit`? 🤔

Reply:
> Maybe because the current trunk-a9a471914d3ebd1093986d3d0a95c2a29b29dca0 version is ahead of the latest [2.80.0](https://github.com/wordpress-mobile/WordPress-FluxC-Android/releases/tag/2.80.0) tag? 🤔

Still, there should be a mention of this dependency in [the logs](https://developer.mend.io/github/wordpress-mobile/WordPress-Android/-/job/0b4829e0-b225-4ff5-a965-ce245457e36d) 😕 

 > Btw, another one, how about dangermattic in addition to having wpmreleasetoolkit? 🤔

True, thanks!

Reply:
Why would `Automattic/dangermattic` match now if you ignore the case in regex for matching dependencies? 🤔 

For example, I am thinking of this `gem 'danger-dangermattic', '~> 1.0'` within the [Gemfile](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/Gemfile#L5).

Reply:
It's very likely related to the complexity connected to our included build setup. This file is browsed by Renovate, but it has difficulties to resolve versions

Reply:
> Still, there should be a mention of this dependency in [the logs](https://developer.mend.io/github/wordpress-mobile/WordPress-Android/-/job/0b4829e0-b225-4ff5-a965-ce245457e36d) 😕

True 🤔

Reply:
> Why would `Automattic/dangermattic` match now if you ignore the case in regex for matching dependencies? 🤔

Because previous regex was matching case-sensitive, so `*A*utomattic/dangermattic` wasn't taken into account.

But now I realized that `Automattic/dangermattic` is in fact a GitHub Action. The `danger-dangermattic` is a gem, without any mention of "Automattic", so we have to add it to whitelist either way

Reply:
After 6d992db, Renovate supports now FluxC and Utils as well - you can verify this in logs https://developer.mend.io/github/wordpress-mobile/WordPress-Android/-/job/018fc95a-01b2-781e-9535-118f8df3c060

Reply:
Oh, nicely done Wojtek! 🥇